src/main.c

```diff
@@ -288,6 +291,46 @@ static uint32_t read_encoder_right(void)
 
 
 /**
+ * @brief Setup for adc: Four injected channels on scan mode for ADC1.
```

Comment:
Just for consistency with other functions, could you add a blank line after the "brief" section? :blush:

Reply:
Also, `ADC` should maybe go in caps. :stuck_out_tongue_winking_eye:

Reply:
ok